internal/client/operation.go

```diff
@@ -158,20 +160,19 @@ func (c *App) Destroy(ctx context.Context, op *pb.Job_DestroyOp) error {
 	return err
 }
 
-func (c *App) Exec(ctx context.Context, op *pb.Job_ExecOp, mon chan pb.Job_State) error {
-	if op == nil {
-		op = &pb.Job_ExecOp{}
-	}
+func (c *App) Exec(ctx context.Context, ec *execclient.Client) (exitCode int, err error) {
```

Comment:
This function was unused previously. Now, the exec cli calls it to trigger the op. This gives the client package here the opportunity to call the internal `setupLocalJobSystem` method to spool up a local runner that can receive the job if the server decides it's necessary.